vllm/engine/arg_utils.py

```diff
@@ -48,6 +49,12 @@ def add_cli_args(
             type=str,
             default=EngineArgs.tokenizer,
             help='name or path of the huggingface tokenizer to use')
+        parser.add_argument(
+            '--revision',
+            type=str,
+            default=None,
```

Comment:
Can you make the default value consistent across the code base? It's `None` here but `"main"` everywhere else.